packages/office-ui-fabric-react/src/components/TextField/TextField.Props.ts

```diff
@@ -95,13 +95,13 @@ export interface ITextFieldProps extends React.HTMLAttributes<HTMLInputElement |
    * Default value of the textfield, if any. Only provide this if the textfield is an uncontrolled component;
    * otherwise, use the "value" property.
    */
-  defaultValue?: string;
+  defaultValue?: string | undefined;
```

Comment:
I think we don't need it, see: https://www.typescriptlang.org/play/index.html#src=interface%20X%20%7B%0D%0A%20%20%20%20foo%3F%3A%20string%3B%0D%0A%7D%0D%0A%0D%0Ainterface%20Y%20%7B%0D%0A%20%20%20%20foo%3A%20string%3B%0D%0A%7D%0D%0A%0D%0A%0D%0Alet%20a%3A%20X%20%3D%20%7B%0D%0A%20%20%20%20foo%3A%20undefined%0D%0A%7D%3B%0D%0A%0D%0Alet%20b%3A%20Y%20%3D%20%7B%0D%0A%20%20%20%20foo%3A%20undefined%0D%0A%7D%3B%0D%0A